### ISSUE TITLE
Discriminator requires_grad=False when training Generator

### ISSUE THREAD

**Issue by njellinas** [caogang/wgan-gp#46]
Hi and thanks for the implementation.

Why the Discriminator parameters' requires_grad property is set to False when training the Generator? Aren't the gradients necessary for backpropagating the error?

Thanks
